Project3/index.html

```diff
@@ -0,0 +1,118 @@
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<link rel="stylesheet" href="styles.css">
+<link href="https://fonts.googleapis.com/css?family=BenchNine:300,400,700|Teko:300,400,500,600,700&display=swap"
+    rel="stylesheet">
+<title>Document</title>
+</head>
+
+<body>
+
+    <form action="">
+        <h2>IDENTITY</h2>
```

Comment:
Text should not be ALL CAPS in your HTML markup -- this should instead be achieved by using `text-transform` in your CSS.